src/data/customer/__test__/route.test.js

```diff
@@ -95,3 +96,52 @@ describe('Retrieve route setting', () => {
     expect('settingRouteSettings' instanceof Array).toBe(true);
   });
 });
+
+describe('Test storeRouteAsync function', () => {
+  const payload = {
```

Comment:
@vdanylov the unit tests that you have added are still failing:

 error { statusCode: 400,
        statusText: 'Bad Request',
        errorMessage: 
         [ { key: '0',
             messages: 'The routes.0.pickup_date field is required.' },
           { key: '1',
             messages: ' The routes.0.route_locations field is required.' } ] }